app/views/songs/_mini_groove_song_card.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="songcard">
+	<span>
+	<div class="coverart">
+		<a href=<%= url %> >
+		<% unless coverart == '' %>
+			<img src=<%= coverart %> />
+		<%end%>
+		</a>
+	</div>
+	<div class="songinfo">
```

Comment:
Be consistent in naming.  In this file your classes are using the smashed-words-together style `songinfo`. In the file below, they are using snakecase `new_song`.  Pick one as a group and stick with it.